index.js

```diff
@@ -60,9 +60,10 @@ module.exports = {
     "react/react-in-jsx-scope": 2,
     "import/extensions": ["error", { js: "never", jsx: "never" }],
     "react/self-closing-comp": 2,
-    "react/sort-comp": 2
+    "react/sort-comp": 2,
+    "react-hooks/rules-of-hooks": "error"
```

Comment:
I forget how default cases work - by omitting an entry for exhaustive-deps, does that effectively disable it?

Reply:
Yeah, it just never includes the rule. I suspect that plugin rules don't have defaults - you need to specify them explicitly.